ui/dashboards/src/components/EmptyDashboard/EmptyDashboard.stories.tsx

```diff
@@ -33,13 +34,31 @@ export default meta;
 type Story = StoryObj<typeof EmptyDashboard>;
 
 /**
- * If no props are passed, the component will use default messaging and action
- * buttons.
+ * When viewing a dashboard, the default actions prompt the user to edit the
+ * dashboard.
```

Comment:
"The default empty dashboard in view mode" << Up to you if like these suggestions for the description